bulbs/content/tasks.py

```diff
@@ -123,7 +123,7 @@ def post_article(content, body, fb_page_id, fb_api_url, fb_token_path, fb_dev_mo
 
 
 def delete_article(content, fb_api_url, fb_token_path):
-    fb_access_token = vault.read(fb_token_path)
+    fb_access_token = vault.read(fb_token_path)['authtoken']
```

Comment:
Best you can do is fail gracefully, by catching the error and raising exception to notify sentry.